StripeConnect/StripeConnect/Source/EmbeddedComponentManager.swift

```diff
@@ -33,7 +33,33 @@ public class EmbeddedComponentManager {
         ComponentAnalyticsClient(client: AnalyticsClientV2.sharedConnect,
                                  commonFields: $0)
     }
+    
+    @_spi(DashboardOnly)
+    var baseURL: URL = StripeConnectConstants.connectJSBaseURL
 
+    @_spi(DashboardOnly)
+    var publicKeyOverride: String? = nil
```

Comment:
does this still need to be `@_spi(DashboardOnly)` since it's now settable in `init`? I think it could be made private.

If we decide to keep it, then I think it will need to be marked public to make it accessible from Dashboard.

Reply:
I removed the annotation but I can't make it private because we need to access it for tests.